source/quick-start.txt

```diff
@@ -0,0 +1,110 @@
+.. _entity-framework-quickstart:
+
+===========
+Quick Start
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/quick-start/overview.rst
+
+Create a MongoDB Cluster
+------------------------
+
+Set Up a Free Tier Cluster in Atlas
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To set up your Atlas Free Tier Cluster required for this guide, complete the guide on 
+:guides:`MongoDB Atlas Setup </atlas/account>`.
+
+After completing the steps in the Atlas guide, you have a new MongoDB
+cluster deployed in Atlas, a new database user, and
+:atlas:`sample datasets loaded </sample-data/>` into your cluster. You also have
+a connection string similar to the following in your copy buffer:
+
+.. code-block:: bash
+
+   "mongodb+srv://<username>:<password>@cluster0.abc.mongodb.net/?retryWrites=true&w=majority"
+
+Set Your Connection String
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the following code at the command prompt to save your MongoDB connection string to an 
+environment variable. This method is safer than including your credentials in your source 
+code. 
+
+.. code-block:: bash
+
+   export MONGODB_URI='<your MongoDB URI>'
+
+.. important::
+
+   Make sure to replace the ``<username>`` and ``<password>`` sections of the connection 
+   string with the username and password of your Atlas user.
```

Comment:
Never mind, I was looking at the guide again and they will need to replace username and password. However, I think it would be helpful to mention recording your database user's credentials somewhere, similar to [these instructions](https://www.mongodb.com/docs/drivers/node/current/quick-start/create-a-connection-string/#update-the-placeholders).